pkg/cli/gettoken/gettoken.go

```diff
@@ -153,7 +154,7 @@ func (o *GetTokenOptions) Run() error {
 
 	tokenSet, err := o.tokenCache.FindByKey(o.tokenCacheDir, tokenCacheKey)
 	if err != nil {
-		return err
+		klog.V(5).Infof("cache not be found")
```

Comment:
that's a wrong assumption, the location can just as well be unreadable and so the rest of the code fails

Reply:
We are ignoring the cache retrieval error. Because if there is any error occurred other than the missed cache, we'll get an error during writing tokens into the cache.

Reply:
When we get file not found, we'll continue authentication. If we get any error other than that, we'll short cut the process by throwing this error.